frontend/catalyst/passes.py

```diff
@@ -13,27 +13,22 @@
 # limitations under the License.
 
 """
-This module contains public API functions that provide control for the
-user to input what MLIR compiler passes to run.
-
-Currently, each pass has its own user-facing decorator. In the future,
-a unified user interface for all the passes will be available.
+This module contains which provides Python decorators
+for enabling and configuring individual Catalyst MLIR compiler passes.
 
 .. note::
 
     Unlike PennyLane :doc:`circuit transformations <introduction/compiling_circuits>`,
-    the QNode itself will not be changed or transformed.
-
-    In other words, circuit inspection tools such as
-    :func:`~.draw` will still
-    display the neighbouring self-inverse gates. However, Catalyst never
-    executes the PennyLane code directly; instead, Catalyst captures the
-    workflow from Python and lowers it into MLIR, performing compiler
-    optimizations at the MLIR level.
-    To inspect the compiled MLIR from Catalyst, use
-    :func:`~.get_compilation_stage`,
-    where ``stage="QuantumCompilationPass"``, and with ``keep_intermediate=True``
-    in the ``qjit`` decorator.
+    the QNode itself will not be changed or transformed by applying these
+    decorators.
+
+    As a result, circuit inspection tools such as :func:`~.draw` will continue
+    to display the circuit as written in Python.
```

Comment:
@paul0403 it's not actually possible to draw a QNode that had `cancel_inverses` applied:

```python
from catalyst.debug import get_compilation_stage
from catalyst.passes import cancel_inverses

dev = qml.device("lightning.qubit", wires=1)

@qjit
@cancel_inverses
@qml.qnode(dev)
def circuit(x: float):
    qml.RX(x, wires=0)
    qml.Hadamard(wires=0)
    qml.Hadamard(wires=0)
    return qml.expval(qml.PauliZ(0))
```

```pycon
>>> print(qml.draw(circuit)(0.54))
NotImplementedError: 
```

Reply:
This works fine though if I comment out the `cancel_inverses`.

Reply:
I think it's because of the newly injected `apply_registered_pass` jax primitive for the pass.  I think for now we can just remove this line in the documentation regarding `draw`, as the cancel-inverse-ed qnode is not supposed to be drawn anyway (the proper inspection tool is always the mlir itself). We can discuss how to draw qjit-ted circuits further in the future if necessary.